src/applications/find-va-forms/tests/actions/index.unit.spec.js

```diff
@@ -51,9 +51,10 @@ describe('Find VA Forms actions', () => {
   });
 
   describe('fetchFormsThunk', () => {
-    const oldWindow = global.window;
+    let oldWindow;
```

Comment:
During a certain unit test execution order, I think `oldWindow` was storing a reference to a stubbed window.